src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt

```diff
@@ -21,89 +21,106 @@ import java.time.temporal.ChronoUnit
 import java.util.Map
```

Comment:
Tidied up tests - ready for additional tests to be added (and added additional tests for agencyId access check)